tests/unit/resources/settings/test_appliance_device_snmp_v3_trap_destinations.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2018) Hewlett Packard Enterprise Development LP
```

Comment:
better to keep it align with the [License](https://github.com/HewlettPackard/python-hpOneView/blob/master/LICENSE)

Reply:
@soodpr - Copyright fixed on commit ceb018d03ebeb53b1fa7de58ff172636b6df7f55